guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -117,97 +110,97 @@ Do not install or update any packages at this stage.
 +
 [options="nowrap"]
 ----
-baseurl=file:///media/sat6/
+baseurl=file:///media/sat6/Satellite
 ----
 
-. Configure the Ansible repository from the ISO file.
+. Configure the {ProjectName} Maintenance repository from the ISO file.
 
-.. Copy the ISO file's repository data file for Ansible packages:
+.. Copy the ISO file's repository data file for {ProjectName} Maintenance packages:
 +
 [options="nowrap"]
 ----
-# cp /media/sat6/ansible/media.repo /etc/yum.repos.d/ansible.repo
+# cp /media/sat6/Maintenance/media.repo /etc/yum.repos.d/satellite-maintenance.repo
```

Comment:
This is a very specific case, and if the idea is to DRY this up, I am lean more towards not doing that here as it could serve to confuse things given this is a special set of paths.

Reply:
@bangelic I'm against attributes here, see my comment in the main thread